ui/pages/add-token/add-token.component.js

```diff
@@ -253,10 +253,8 @@ class AddToken extends Component {
     } = this.state;
 
     const { chainId } = this.props;
-    const blockExplorerTokenLink = createTokenTrackerLinkForChain(
-      customAddress,
-      chainId,
-    );
+    const blockExplorerTokenLink = getTokenTrackerLink(customAddress, chainId);
```

Comment:
Here in order for the custom block explorers to work you'll need to get RPCPrefs from state and pass them in as the 5th argument here... I made a stupid function signature (probably need to change it). See line 51 in the token-asset.js file [here](https://github.com/MetaMask/metamask-extension/pull/11017/files) for reference.

Reply:
@NiranjanaBinoy ^